registry/storage/driver/gcs/gcs.go

```diff
@@ -165,10 +166,7 @@ func FromParameters(ctx context.Context, parameters map[string]interface{}) (sto
 			return nil, err
 		}
 		ts = jwtConf.TokenSource(ctx)
-		gcs, err = storage.NewClient(ctx, option.WithCredentialsFile(fmt.Sprint(keyfile)))
-		if err != nil {
-			return nil, err
-		}
+		options = append(options, option.WithCredentialsFile(fmt.Sprint(keyfile)))
```

Comment:
The `fmt.Sprint()` everywhere make my eyes bleed 😂

Reply:
The whole config file needs rework in v4. I'm pretty sure I saw an issue opened by @stevvooe last century. It's somewhere in the backlog

Reply:
Yeah, I spotted it in your changed lines, but then saw it was all over the place, so don't bother for this PR